testfixtures/__init__.py

```diff
@@ -1,16 +1,19 @@
 # Copyright (c) 2008-2014 Simplistix Ltd
 # See license.txt for license details.
 
+
 class singleton(object):
     def __init__(self, name):
         self.name = name
+
     def __repr__(self):
         return '<%s>' % self.name
     __str__ = __repr__
-    
+
 not_there = singleton('not_there')
 
-from testfixtures.comparison import Comparison, StringComparison, RoundComparison, compare, diff
+from testfixtures.comparison import (Comparison, StringComparison,
+                                     RoundComparison, compare, diff)
 from testfixtures.tdatetime import test_datetime, test_date, test_time
```

Comment:
Your change does noe make diffs easier, having one imported object per line would, but then we're back to having 30 lines at the top of each file to cover the imports...